File: admin-common/src/main/java/com/ktko/admin/domain/manager/mapper/ManagerMapper.java
package com.ktko.admin.domain.manager.mapper;

import com.ktko.admin.domain.manager.entity.Manager;
import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Param;
import org.apache.ibatis.annotations.Select;
import org.springframework.stereotype.Repository;

import java.util.List;

@Mapper
@Repository
public interface ManagerMapper {

    @Select("SELECT * FROM admin_manager")
    public List<Manager> getManagers();

    @Select("SELECT * FROM admin_manager" +
            "WHERE email = #{email} AND password = #{password}")
    public Manager getManager(@Param("email") String eMail, @Param("password") String password);
}

File: admin-common/src/main/java/com/ktko/admin/domain/manager/mapper/ManagerMapper.kt
package com.ktko.admin.domain.manager.mapper

import com.ktko.admin.domain.manager.entity.Manager
import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Param
import org.apache.ibatis.annotations.Select
import org.springframework.stereotype.Repository

@Mapper
@Repository
interface ManagerMapper {


    @Select("""<script>
        SELECT * FROM admin_manager
    </script>""")
    fun getManagers(): List<Manager>

    @Select("""<script>
       SELECT * FROM admin_manager WHERE email = #{email} AND password = #{password}
    </script>""")
    fun getManager(@Param("email") eMail: String, @Param("password") password: String): Manager
}
